R/tidyverse.R

```diff
@@ -249,6 +249,34 @@ spread.sf <- function(data, key, value, fill = NA, convert = FALSE, drop = TRUE,
 		drop = drop, sep = sep), sf_column_name = attr(data, "sf_column"))
 }
 
+#' @name tidyverse
+#' @export
+#' @examples
+#' library(tidyr)
+#' nc %>% select(SID74, SID79) %>% gather("VAR", "SID", -geometry) %>% summary()
+pivot_longer.sf <- function(data, cols, names_to = "name", names_prefix = NULL,
```

Comment:
I suggest to replace all the parameters that you want to pass on untouched as ... , unless you want to change their default. This would spare you (and me) the work of documenting them, and keeping track of changes in upcoming tidyr versions.